### DISCUSSION TITLE
Panic thrown by ropey crate after idling overnight

### DISCUSSION THREAD

**Discussion by vosov** [helix-editor/helix#8846] [General]
### Summary

I left helix open idling overnight after doing some editing. When I got back to it today the application panicked. The crash happened when I focused the helix window and made the first keypress. The error message dumped in the terminal was
```
thread 'main' panicked at /build/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ropey-1.6.1/src/slice.rs:360:41:
called `Result::unwrap()` on an `Err` value: Char index out of bounds: char index 14053, Rope/RopeSlice char length 2194
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
helix  62.31s user 5.01s system 0% cpu 13:01:39.95 total
```

### Reproduction Steps

I haven't been able to reproduce the error.

### Helix log

Nothing was written to the log file between my activity yesterday and the crash today.

### Platform

Linux

### Terminal Emulator

xterm

### Installation Method

pacman

### Helix Version

23.10

**Comment by pascalkuthe** [comment]
this crash is very generic and could be caused by anything. Withoult a reproduction case this is not actionable
